### PR TITLE
feat(onboarding-task-list): reskin and rewrite

### PR DESCRIPTION
**Pull request by doralchan**
**Before:**
Screen Shot 2021-02-16 at 11 45 21 AM

**After:**
onboarding

### CODE REVIEW

File: src/sentry/static/sentry/app/components/onboardingWizard/taskConfig.tsx [hunk 1]
```diff
@@ -240,19 +232,18 @@ export function getMergedTasks(organization: Organization) {
...
 const PulsingIndicator = styled('div')`
   ${pulsingIndicatorStyles};
+  margin-right: ${space(1)};
 `;
 
 const EventWaitingIndicator = styled(p => (
   <div {...p}>
-    {t('Waiting for first event')}
     <PulsingIndicator />
+    <span>{t('Waiting for event')}</span>
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/onboardingWizard/taskConfig.tsx:241]: do you need this `span` here?

**Review by priscilawebdev — APPROVED** [review]
tested it locally and it looks great!! 🎉

I would just adjust the `TaskSidebarPanel`  to fit the width on smaller devices. Please see:

image

but I think we can do that in a follow-up PR.

Great work @doralchan 👏

File: src/sentry/static/sentry/app/components/onboardingWizard/progressHeader.tsx [hunk 2]
```diff
@@ -14,45 +14,49 @@ type Props = {
...
         font-size: 26px;
-        color: ${theme.gray300};
+        color: ${theme.textColor};
       `}
     />
-    <HeadingText>
-      <h4>{t('Setup Sentry')}</h4>
-      <p>{t('Complete these tasks to take full advantage of Sentry in your project')}</p>
-    </HeadingText>
+    <HeaderTitle>{t('Quick Start')}</HeaderTitle>
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/onboardingWizard/progressHeader.tsx:30]: maybe we can name it only `Title`?

File: src/sentry/static/sentry/app/components/onboardingWizard/task.tsx [hunk 3]
```diff
@@ -81,26 +81,28 @@ function Task({router, task, onSkip, onMarkComplete, forwardedRef, organization}
...
-            title={t('No user was associated with completing this task')}
-          >
-            <TaskBlankAvatar round />
-          </Tooltip>
-        )}
-      </ItemComplete>
+      <TaskCard ref={forwardedRef} onClick={handleClick}>
+        <CompleteTitle>
+          <StatusIndicator>
+            {task.status === 'complete' && <CompleteIndicator />}
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/onboardingWizard/task.tsx:87]: maybe we should destructure these props?

File: src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx [hunk 4]
```diff
@@ -131,6 +135,8 @@ const SidebarPanelBody = styled('div')<{hasHeader: boolean}>`
   flex-direction: column;
   flex-grow: 1;
   overflow: auto;
+  padding: 100px 30px;
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx:138]:
```suggestion
  padding: 100px ${space(4)};
```

File: src/sentry/static/sentry/app/components/onboardingWizard/taskConfig.tsx (hunk 1 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/components/onboardingWizard/taskConfig.tsx]: Removed

File: src/sentry/static/sentry/app/components/onboardingWizard/task.tsx (hunk 3 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/components/onboardingWizard/task.tsx:87]: I can try to do this in another PR 👍

**Pull request merged**